src/applications/vaos/api/index.js

```diff
@@ -2,8 +2,6 @@ import moment from 'moment';
 import { apiRequest } from 'platform/utilities/api';
 import environment from 'platform/utilities/environment';
 
-// This wil go away once we stop mocking api calls
-const TEST_TIMEOUT = navigator.userAgent === 'node.js' ? 1 : null;
```

Comment:
This wasn't being used anymore after updating `getSupportingSites()` so I removed it.